app/views/issue_templates/show.html.erb

```diff
@@ -1,80 +1,90 @@
-<%= link_to_if_authorized l(:button_delete), 
-  {:controller => 'issue_templates', :action => 'destroy', 
-	:id => @issue_template, :project_id => @project}, 
-    :data => {:confirm => l(:text_are_you_sure)}, :method => :post, :class => 'icon icon-del' %>
-<%= link_to(l(:label_list_templates),
-              {:controller => :issue_templates,
-              :action => :index,
-              :project_id => @project}, :class => 'template') %>  
+  <%= link_to_if_authorized l(:button_edit),
+                            { controller: 'issue_templates', action: 'edit', id: @issue_template,
+                              project_id: @project },
+                            class: 'icon icon-edit', accesskey: accesskey(:edit),
+                            onclick: "$('#edit-issue_template').show(); return false;" %>
+  <%= link_to_if_authorized l(:button_delete),
+                            { controller: 'issue_templates', action: 'destroy',
+                              id: @issue_template, project_id: @project },
+                            data: {confirm: l(:text_are_you_sure)}, method: 'post', class: 'icon icon-del' %>
+  <%= link_to(l(:label_list_templates),
+              { controller: 'issue_templates',
+                action: 'index',
+                project_id: @project }, class: 'template') %>
 </div>
 
-<h2 class="template"><%= l(:issue_templates) %>: #<%= @issue_template.id %> <%= @issue_template.title %> <%= avatar(@issue_template.author, :size => "24") %></h2>
+<h2 class="template">
+  <%= l(:issue_templates) %>: #<%= @issue_template.id %> <%= @issue_template.title %>
+  <%= avatar(@issue_template.author, size: '24') %>
+</h2>
 
 <% if authorize_for('issue_templates', 'edit') %>
-<div id="edit-issue_template" style="display:none;">
-<%= labelled_form_for :issue_template, @issue_template, :url => { :controller => 'issue_templates', :action => 'edit', :project_id => @project, :id => @issue_template },
-       :html => { :id => 'issue_template-form',
-       :class => nil,
-       :multipart => false } do |f| %> 
-<%= render :partial => 'form', :locals => { :f => f } %>
-<%= submit_tag l(:button_save) %>
-<% end %>
-  <%= preview_link url_for(:controller => 'issue_templates', :action => 'preview',
-                           :id => @issue_template),
-                   'issue_template_description', target='preview' %> |
-<%= link_to l(:button_cancel), {:action => 'index'}, :onclick => 'Element.hide("edit-issue_template"); return false;' %>
-<div id="preview" class="wiki"></div>
-<hr/>
-</div>
+    <div id="edit-issue_template" style="display:none;">
+      <%= labelled_form_for :issue_template, @issue_template,
+                            url: { controller: 'issue_templates', action: 'edit',
+                                   project_id: @project, id: @issue_template },
+                            html: { id: 'issue_template-form', class: nil, multipart: false } do |f| %>
+          <%= render partial: 'form', locals: { f: f } %>

```

Comment:
__[rails_best_practices]__
* simplify render in views [:ref](http://rails-bestpractices.com/posts/2010/12/04/simplify-render-in-views/)